packages/evolution-frontend/src/components/survey/Question.tsx

```diff
@@ -392,7 +394,28 @@ export class Question<CustomSurvey, CustomHousehold, CustomHome, CustomPerson> e
                     )}
                 </InputWidgetWrapper>
             </div>
-        );
+        )];
+
+        switch(this.props.join) {
+            case "next":
+                content.push(
+                <div
+                    key="postJoin"
+                    style={{marginTop: '-40px', marginBottom: '-40px', paddingTop: '20px', paddingBottom: '20px'}}
```

Comment:
next and previous are identical?

Reply:
Yes, the html code pulls from the top and bottom question evenly to join them, so it is only a matter of putting it before or after the question.

Reply:
oh ok! one is push the other unshift, ok. Maybe worth a comment in the code then.